inventory/sample/group_vars/all.yml

```diff
@@ -27,3 +27,13 @@ metal_lb_controller_tag_version: "v0.12.1"
 
 # metallb ip range for load balancer
 metal_lb_ip_range: "192.168.30.80-192.168.30.90"
+
+# If managing more then one config file / enviornment
+# pull file to custom local location and change
+# change default to a custom option
+# Example:
+# local_k3s_config_filename: "homelab"
+# local_k3s_cluster_context: "homelab"
+
+local_k3s_config_filename: "config.k3s.downloaded"
```

Comment:
Are these required now?  If you omit them will it just skip copying?

Reply:
Is this optional?  If this is commented out will the task fail?

Reply:
Hello Guys
Just a little tip how I did it in my personal playbook.
You can use fetch to store the k3s config in your ansible role directory, just if you are gonna do It like I did, check that the task only gets executed on the first master. Other wise the flat argument is the reason it is gonna overwrite itself from each node.

```
- name: Copy cluster config to Ansible master
  fetch:
    src: /etc/rancher/k3s/k3s.yaml
    dest: "{{ role_path }}/files/cluster.yaml"
    flat: yes
```